packages/postgres/pg-queue.ts

```diff
@@ -14,12 +14,42 @@ import {
   asExpressions,
   Deferred,
   Job,
+  fullReindexBatchTimeoutSeconds,
+  normalizeFullReindexBatchSize,
+  FROM_SCRATCH_JOB_TIMEOUT_SEC,
+  normalizeFullReindexCooldownSeconds,
 } from '@cardstack/runtime-common';
 import type { PgAdapter } from './pg-adapter';
 import * as Sentry from '@sentry/node';
 
 const log = logger('queue');
-const MAX_JOB_TIMEOUT_SEC = 20 * 60;
+
+function parsePositiveInteger(value: string | undefined): number | undefined {
+  if (value == null) {
+    return undefined;
+  }
+  let parsed = Number(value);
+  if (!Number.isFinite(parsed) || parsed < 1) {
+    return undefined;
+  }
+  return Math.floor(parsed);
+}
```

Comment:
This seems excessively defensive. Can we trust ourselves to set PG_QUEUE_MAX_JOB_TIMEOUT_SEC to something that will parse cleanly as a positive integer?